server/dash/dashboards/neighborhood_recent.py

```diff
@@ -174,19 +181,58 @@ def update_table(selected_council):
     Input("council_list", "value")
 )
 def update_text(selected_council):
+    """Dash Callback Function to update the indicator cards at the top of the dashboard.
+
+    This function takes the selected neighborhood council (nc) value from the "council_list" dropdown and 
+    outputs the values for the number of new requests, number of closed requests, and net change in requests
+    (i.e. # closed requests - # new requests) for visualizations on the indicator visuals in the layout. The 
+    corresponding IDs of the indicator visuals are "created_txt", "closed_txt", and "net_txt".
+
+    Typical usage example (using net_txt as example, created_txt and closed_txt are similar):
+
+        html.Div(
+            [html.H2(id="net_txt"), html.Label("Net Change")],
+            className="stats-label"
+        )    
+    """
     create_count = df.query(f"councilName == '{selected_council}' and createdDate >= '{start_date}'")['srnumber'].count()  # noqa
     close_count = df.query(f"councilName == '{selected_council}' and closedDate >= '{start_date}'")['srnumber'].count()  # noqa
-    return create_count, close_count, create_count - close_count
+
+    # This check is to ensure data quality issues don't flow downstream to the dashboard (i.e., closed requests exist without any new requests).
+    if create_count == 0 and close_count > 0:
+        return 0, 0, 0
+    else:
+        return create_count, close_count, create_count - close_count
 
 
 @app.callback(
     Output("graph", "figure"),
     Input("council_list", "value")
 )
 def update_figure(selected_council):
+    """Dash Callback Function to update the Request Type Line Chart at the middle of the dashboard.
+
+    This function takes the selected neighborhood council (nc) value from the "council_list" dropdown and 
+    outputs the request type line chart that shows the trend of of different requests types over
+    the time range of the data available in the selected neighborhood conucil. The line chart will
+    show up inside dcc.Graph object as long as id "graph" is passed in.
+
+    Typical usage example:
+
+        html.Div(
+                    dcc.Graph(id='graph', 
+                    ...
+                )
+    """
```

Comment:
please add `Args` and `Returns` sections

Reply:
Thanks for the suggestion - resolved.